docs/cristin.ttl

```diff
@@ -103,22 +112,23 @@
          """ .
 
 :FAGFELT_Class
-    a       :CristinClass;
-    rdfs:comment """The scientific field in which the book is published""" ;
+    a               :CristinClass ;
+    rdfs:comment    """The scientific field in which the book is published""" ;
```

Comment:
NB: Is also used for single book chapters (BOK_RAPPORT_DEL). The inheritance rules between book and chapter are somewhat arcane.